hardware/asic/skywater/config.tcl

```diff
@@ -7,7 +7,8 @@ set ::env(VERILOG_FILES) "\
     designs/system/src/system.v\
     designs/system/src/boot_ctr.v\
     designs/system/src/int_mem.v\
-    designs/system/src/iob_dp_ram.v\
+    designs/system/src/iob_sp_ram_be.v\
```

Comment:
You should try to pass a list of VSRC files instead of having a hardcoded list.

Reply:
how I will pass the VSRC to config.tcl file ?

Reply:
Try to see how it is done for the .tcl script for FPGAs and umc130. Those platforms use .tcl scripts that read VSRC as an argument.